lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.html

```diff
@@ -1,6 +1,8 @@
 <mat-card class="adf-start-process">
-
-    <mat-card-title class="adf-title">{{'ADF_CLOUD_PROCESS_LIST.ADF_CLOUD_START_PROCESS.FORM.TITLE' | translate}}
+    <mat-card-title
+        *ngIf="showTitle"
```

Comment:
When would we need to hide the title?

Reply:
it is required by this story [AAE-6345](https://alfresco.atlassian.net/browse/AAE-6345)

i.e. `"Remove “Start process” title"`